src/NovaDependencyContainer.php

```diff
@@ -154,15 +154,21 @@ public function resolveForDisplay($resource, $attribute = null)
             }
 
             if (array_key_exists('value', $dependency)) {
-                if($dependency['value'] == $resource->{$dependency['property']}) {
+                if(is_array($resource) && $dependency['value'] == $resource[$dependency['property']]) {
```

Comment:
This appears to be entirely unrelated to adding css styles. I would say that you should split css and whatever you are fixing here into 2 PRs with additional clarity on what the updates here are doing in that second PR.